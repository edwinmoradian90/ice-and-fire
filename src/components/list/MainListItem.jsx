import React from 'react'
import PropTypes from 'prop-types'
import { StyledLink, Text } from '../../utils/styledComponents/main'

function MainListItem(props) {
    const item = props.item
    return (
        <>
            {

                <StyledLink to={item} className='listItem' >
                    <Text
                        className='itemTitle'
                        hColor='red'
                    >
                        {item}
                    </Text>
                </StyledLink >

            }
        </>
    )
}

MainListItem.propTypes = {
    item: PropTypes.string.isRequired,
}

export default MainListItem