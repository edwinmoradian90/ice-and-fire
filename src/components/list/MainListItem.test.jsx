import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import MainListItem from './MainListItem';

test('renders correctly and matches snapshot', () => {
    const component = renderer.create(
        <Router>
            <MainListItem />
        </Router>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});