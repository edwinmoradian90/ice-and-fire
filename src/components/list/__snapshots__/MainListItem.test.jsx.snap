// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly and matches snapshot 1`] = `
<a
  className="sc-AxjAm irMvbk listItem"
  href=""
  onClick={[Function]}
>
  <div
    className="sc-AxhUy ilDfEv itemTitle"
  />
</a>
`;
